src/main/java/com/byznass/tiolktrack/dao/plsql/PlsqlGpsProvider.java

```diff
@@ -0,0 +1,30 @@
+package com.byznass.tiolktrack.dao.plsql;
+
+import com.byznass.tiolktrack.kernel.dao.GpsProvider;
+import com.byznass.tiolktrack.kernel.dao.NoGpsWithIdException;
+import com.byznass.tiolktrack.kernel.model.Gps;
+import com.byznass.tiolktrack.kernel.model.Location;
+
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+public class PlsqlGpsProvider implements GpsProvider {
+
+	@Override
+	public Gps getGpsById(String gpsId) throws NoGpsWithIdException {
+
+		//TODO (TT-20) implement this provider
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3AS1135)